requirements.txt

```diff
@@ -0,0 +1,2 @@
+ops >=2,<3
+jenkinsapi>=0.3
```

Comment:
What is the reason for using this compared to, for example, this: https://pypi.org/project/python-jenkins/?

Reply:
I initially thought that library was pointing to an archived library https://github.com/ceph/python-jenkins. I just realized that's a fork. Will replace with that library!